Python/other/Huffman_coding.py

```diff
@@ -0,0 +1,94 @@
+'''Python code to implement HUffman encoding 
```

Comment:
This is Huffman encoding algorithm only, edit the docstring. Codes contributed in python are expected to follow our [Python code template](https://github.com/TesseractCoding/NeoAlgo/blob/master/Python/Code_Template.md) to ease the reviewing process and maintain the quality of contributed code.

Reply:
Surely I will look into the matter and make the changes.